### PR TITLE
Feature sprint09 tijn searchable d3d and pp

### PR DESCRIPTION
**Pull request by tschmits**

### CODE REVIEW

File: app/scripts/components/modeldetails.js [hunk 1]
```diff
@@ -39,39 +39,39 @@ var exports = (function () {
       dateCreatedText: {
         cached: false,
         get: function () {
-          var d = new Date(this.activeModel.data.date_created);
+          var d = new Date(_.get(this.activeModel, ".data.date_created"), "");
```
Comment (SiggyF) [inline app/scripts/components/modeldetails.js]: I think the first `.` should be removed in `".data.date_created"`

File: app/scripts/components/searchdetails.js [hunk 2]
```diff
@@ -47,14 +50,16 @@ var exports = (function () {
       };
 
       // get search templates
-      Promise.all([fetchUsers(), fetchSearchTemplate()])
+      Promise.all([fetchUsers(), fetchSearchTemplate(), fetchVersions()])
         .then((jsons) => {
           var users = jsons[0];
           var template = jsons[1];
+          var versions = jsons[2];
```
Comment (SiggyF) [inline app/scripts/components/searchdetails.js]: You could use [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment). I believe our code is transpiled by babel, so it should be compatible. `[users, template, versions] = jsons;`

File: app/scripts/components/searchdetails.js [hunk 3]
```diff
@@ -235,6 +239,22 @@ var exports = (function () {
...
+
+        return _.omit(versions, filter);
+      },
+      niceVersionTitle: function (id) {
+        var mappings = {
+          "delft3d_version": "Delft3D version",
+          "SVN_REV": "SVN revision"
+        };
+
+        return _.get(mappings, "." + id, id);
```
Comment (SiggyF) [inline app/scripts/components/searchdetails.js]: I'm not sure if this `.` will work. I can't find it documented.

File: app/templates/search-details.html [hunk 4]
```diff
@@ -67,6 +67,29 @@
...
 
+          <div class="panel panel-default panel-search">
+            <div class="panel-heading panel-collapse-header" data-toggle="collapse" data-parent="#template-versions" href="#template-versions" aria-expanded="true" aria-controls="template-collapse">
+              Versions
+            </div>
+            <div class="panel-body panel-collapse collapse" role="tabpanel" aria-labelledby="template-versions" id="template-versions">
+              <div class="form-group" v-for="(key, values) in filterVersions(versions)">
+                <select
+                   v-model="selectedVersions[key]"
+                   id="{{ key }}"
```
Comment (SiggyF) [inline app/templates/search-details.html]: `{{ }}` is not allowed in attributes starting from version 2.0, so please use `v-bind:id="key"`.

File: app/templates/search-details.html [hunk 5]
```diff
@@ -67,6 +67,29 @@
...
+                   class="select-picker form-control"
+                   title="{{ niceVersionTitle(key) }}"
+                   multiple
+                   data-selected-text-format="count > 3"
+                   data-actions-box="true"
+                   data-container="body">
+                   <template v-for="v in values">
+                    <option data-content="{{ v }}">{{ v }}</option>
```
Comment (SiggyF) [inline app/templates/search-details.html]: `v-bind:data-content="v"` or :data-content="v"

File: gulpfile.babel.js [hunk 6]
```diff
@@ -27,7 +27,7 @@ const reload = browserSync.reload;
 var args = require("yargs").argv;
 // Server used for serving remote url"s
 // "http://136.231.10.175:8888";
-var apiServer = "https://delft3dgt-nginx";
+var apiServer = "http://delft3dgt-django:8000";
```
Comment (SiggyF) [inline gulpfile.babel.js]: for debugging? do you want to check this in?

File: app/scripts/components/searchdetails.js (hunk 2 above)
Comment (tschmits) [inline app/scripts/components/searchdetails.js]: Nice!

**Pull request merged**